docs/docs/glossary.md

```diff
@@ -0,0 +1,112 @@
+---
+title: Glossary
+lang: en-US
+meta:
+  - name: keywords
+    content: pomerium identity-access-proxy beyondcorp zero-trust reverse-proxy ztn zta
+description: A quick reference of commonly used terms.
+---
+
+# Glossary
+
+Pomerium's documentation use a lot of terminology specific to the networking and security space. This glossary defines common terms readers may be unfamiliar with. If you come across an unfamiliar term not listed in this page, please let us know in our [Discuss support forum][support] and we'll add it.
+
+[[toc]]
+
+## General
+
+### Access Token
+This is a general term referring to a string issued by an identifying service like an [identity provider] that validates the holder to have a specific set of permissions. Most of the access tokens discussed in our docs are [JWTs].
+
+### Identity Provider
+An identity provider (**IdP**) is used to [authenticate] a user, i.e. confirm their identity. Pomerium uses external IdPs to better integrate into existing environments, and to achieve strong separation of services.
+
+### Identity-aware Proxy
+A [proxy](https://en.wikipedia.org/wiki/Proxy_server) is an intermediate service between a one or more clients or servers. Most of the proxies discussed in our docs are technically [reverse proxies](https://en.wikipedia.org/wiki/Reverse_proxy), sitting between one or more severs and all clients, providing a single point of ingress into a system.
+
+An identity-aware proxy can provide contextual access to specific services based on the identity of the client. In Pomerium's case, identity is provided by the client in the form of a [JWT] issued by the [identity provider].
+
+### JavaScript Object Notation
+Commonly shortened to **JSON**, [JavaScript object notation](https://en.wikipedia.org/wiki/JSON) is a common format used to represent and share structured sets of data as arrays of key-value pairs.
+
+### JSON Web Key Sets
+Usually abbreviate as **JWKS**, this is a [JSON]-formatted set of one or more keys provided by a trusted issuer and used by service to verify [JWTs] provided by a client.
+
+### JSON Web Token
+Often referred to as **JWTs**, a JSON web token is a [JSON]-formatted string provided to a user by an [identity provider], which validates the user's identity to subsequent services (such as an [identity-aware-proxy]).
+
+### Namespace
+"Namespaces" is an over-saturated term, having different meanings in different contexts. [Pomerium Enterprise][pom-namespace] uses Namespaces to provide separation of access and control to [routes]. Kubernetes uses their [namespaces][k8s-namespace] to isolate groups of resources within a cluster.
+
+### Perimeter
```

Comment:
We should link to the wikipedia https://en.wikipedia.org/wiki/Perimeter_security

Reply:
agree